infrastructure-playbooks/shrink-mds.yml

```
@@ -0,0 +1,110 @@
+---
+# This playbook removes the Ceph MDS from your cluster.
+#
+# Use it like this:
+# ansible-playbook shrink-mds.yml -e mds_to_kill=ceph-mds01
+#     Prompts for confirmation to shrink, defaults to no and
+#     doesn't shrink the cluster. yes shrinks the cluster.
+#
+# ansible-playbook -e ireallymeanit=yes|no shrink-mds.yml
+#     Overrides the prompt using -e option. Can be used in
+#     automation scripts to avoid interactive prompt.
+- name: gather facts and check the init system
+  hosts:
+    - "{{ mon_group_name | default('mons') }}"
+    - "{{ mds_group_name | default('mdss') }}"
+  become: true
+  tasks:
+    - debug: msg="gather facts on all Ceph hosts for following reference"
```

you must import the role ceph-defaults here
```suggestion
    - import_role:
        name: ceph-defaults
    - debug: msg="gather facts on all Ceph hosts for following reference"
```

---

Should `ceph-facts` too move here?

---

not needed